force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -762,27 +678,23 @@ export default class rd2EntryForm extends LightningElement {
             }
         } catch (error) {}
 
-        this.setSaveButtonDisabled(false);
+        this.enableSaveButton();
     }
 
     /***
      * @description Handle error and disable the Save button
      */
     handleError(error) {
         this.error = error && error.detail ? error : constructErrorMessage(error);
-
-        this.setSaveButtonDisabled(true);
+        this.disableSaveButton();
     }
 
-    /***
-     * @description Handle component display when an error occurs
-     * @param isDisabled: Indicates if the Save button should be disabled
-     */
-    setSaveButtonDisabled(isDisabled) {
-        const disableBtn = !!(isDisabled && isDisabled === true);
+    disableSaveButton() {
+        this.isSaveButtonDisabled = true;
+    }
 
-        this.isLoading = disableBtn;
```

Comment:
I think this change is preventing error messages from appearing, the Loading spinner stays visible, hiding the error.

Reply:
I think we may need to add a line to `handleError` to disable the loading spinner to fully replicate the former behavior of `setSaveButtonDisabled`. Will see if that fix works and get a test in there for it if so.